apps/web/src/components/PrivateRoute/PrivateRoute.tsx

```diff
@@ -6,7 +6,7 @@ import { useAppState } from '../../state';
 export default function PrivateRoute({ children, ...rest }: RouteProps) {
   const { isAuthReady, user } = useAppState();
 
-  const renderChildren = user || !process.env.REACT_APP_SET_AUTH;
+  const renderChildren = user;
```

Comment:
This looks kinda funny. Wonder if we could just do:

`if (!user && !isAuthReady)`